htdocs/main.inc.php

```diff
@@ -738,7 +738,7 @@ function analyseVarsForSqlAndScriptsInjection(&$var, $type)
 		// Verification security graphic code
 		if ($test && GETPOST("username", "alpha", 2) && !empty($conf->global->MAIN_SECURITY_ENABLECAPTCHA) && !isset($_SESSION['dol_bypass_antispam'])) {
 			$sessionkey = 'dol_antispam_value';
-			$ok = (array_key_exists($sessionkey, $_SESSION) === true && (strtolower($_SESSION[$sessionkey]) === strtolower(GETPOST('code', 'restricthtml'))));
+			$ok = (array_key_exists($sessionkey, $_SESSION) === true && $_SESSION[$sessionkey] === GETPOST('code', 'restricthtml'));
```

Comment:
Goal of captchca is to add a layer test difficult tobread for non human, not to increase the entropy of possible combination (this is the goal of password). For such needs, beeing case sensitive is useless.